fiscal_epos_print/static/lib/fiscalprint/fiscalprint.js

```diff
@@ -0,0 +1,1435 @@
+﻿//
+// ePOS-Print and Fiscal Print API
+//
+// Version 1.0.0
```

Comment:
Sulla 12 è stato aggiornato alla `Version 1.1.0`. Sarebbe da fare anche qua

Reply:
Ok! Faccio subito! Tra l'altro anche la libreria di epson ha introdotto la ``function decodeFpStatus(add_info.fpStatus)`` ma riporta gli stati in italiano